examples/esp32/esp32.rs

```diff
@@ -57,7 +57,7 @@ fn main() -> anyhow::Result<()> {
         use micro_rdk::esp32::analog::Esp32AnalogReader;
         use micro_rdk::esp32::base::Esp32WheelBase;
         use micro_rdk::esp32::board::EspBoard;
-        use micro_rdk::esp32::motor::MotorEsp32;
+        use micro_rdk::esp32::motor::ABMotorEsp32;
```

Comment:
Will these namings be considered breaking in the future?

Reply:
Once we have config consumption fully implemented and rely on model names like in RDK, they won't be